Updated instructions for ALEKS.

npthiery opened the pull request:
Hi Brian,

I made a minor update to the ALEKS tool json description.

Thanks for setting up that very nice LTI page.

Best,

Nico

whitmer committed:
configuring xml links, fix khan link

whitmer committed:
fix icons to not use .ico files

whitmer committed:
setting tweaks

whitmer committed:
show copy/paste textarea for embedding content if not in a supported lms

whitmer committed:
add merlot as possible tool

whitmer committed:
added schooltube tool

whitmer committed:
spelling city tool

whitmer committed:
add schooltube to collections tool

whitmer committed:
update educreations list

whitmer committed:
tweaks

whitmer committed:
bad link

whitmer committed:
remove real-world examples on extensions page that belong on home page

whitmer committed:
programr and google art project data sets

whitmer committed:
analytics for all tools

whitmer committed:
track even miscode

whitmer committed:
added funbrain and filament games tools

whitmer committed:
tweak analytics event tracking

whitmer committed:
MIT license

whitmer committed:
storify tool

whitmer committed:
cengage tool

whitmer committed:
pinterest tool

whitmer committed:
pinterest preview width

whitmer committed:
mistakes? what?

whitmer committed:
inigral tool

whitmer committed:
better support for data-only tools

whitmer committed:
canvas app store helper js and css

whitmer committed:
typo

whitmer committed:
heroku troubleshoot

whitmer committed:
heroku troubleshoot

whitmer committed:
read json file instead of requesting it

whitmer committed:
track tool launches in ganalytics

whitmer committed:
verbiage

whitmer committed:
recently added filter for tools

whitmer committed:
slightly taller tools list

whitmer committed:
canvas already uses titanpad

whitmer committed:
verbiage

whitmer committed:
longer descriptions, uses updates

whitmer committed:
added wiktionary tool

whitmer committed:
smarterer tool

whitmer committed:
remove old new ones

whitmer committed:
studyegg tool

whitmer committed:
hoot.me tool, ocw and cnx search endpoints

whitmer committed:
fix usa today tool

whitmer committed:
connexions and ocw search tools

whitmer committed:
connexions is not performant, also fixed a search bug

whitmer committed:
typo

whitmer committed:
better tooltips and sage tool

whitmer committed:
tweaks

whitmer committed:
disable pinterest until the api gets figured out

whitmer committed:
mahara, wordpress, q2a, status.net tools

whitmer committed:
added missing tool icons

whitmer committed:
add a tool id to config xml

whitmer committed:
links to json and jsonp data sets

whitmer committed:
ck-12 tool

whitmer committed:
mathalicious tool

whitmer committed:
atom feed to tools list

whitmer committed:
manga high tool

whitmer committed:
typo

whitmer committed:
college board tool

whitmer committed:
add previews to google art project

whitmer committed:
fix back button in tools, fix mathalicious tool

whitmer committed:
add break out into separate files, add twitter login, start on handlebars refactor

whitmer committed:
paginated tools get, now using new api

whitmer committed:
ratings part 1

whitmer committed:
ratings part 2

whitmer committed:
ratings part 3

whitmer committed:
a little more ims love

whitmer committed:
fix ratings showing on index page

whitmer committed:
add identity bar for all pages

whitmer committed:
locally cached twitter icon

whitmer committed:
iframe embed for educreations

whitmer committed:
ui cleanup, fix a couple bugs

whitmer committed:
fix labels

whitmer committed:
update extensions page to be more readable

whitmer committed:
remove unnecessary and unused launch urls

whitmer committed:
sanitize user inputs

whitmer committed:
allow graph tool to be inserted as module links

whitmer committed:
support for oembed for link selection

whitmer committed:
add link selection to embed tools

whitmer committed:
don't require handlebars gem in prod env

whitmer committed:
don't require sqlite for prod env

whitmer committed:
troubleshoot

whitmer committed:
troubleshoot

whitmer committed:
fix requires for heroku

whitmer committed:
fix for broken spec

whitmer committed:
layout tweak

whitmer committed:
don't trust local storage for user session tracking

whitmer committed:
api for listing app categories

whitmer committed:
cancel button for comments

whitmer committed:
don't set optional fields if passed as empty string

whitmer committed:
analytics tweak

whitmer committed:
fix quizlet and slideshare

whitmer committed:
clean up navigation

whitmer committed:
app list filtering

whitmer committed:
better filtering

whitmer committed:
fix spacing on voters

whitmer committed:
api documentation

whitmer committed:
change twitter styling

whitmer committed:
https twitter js

whitmer committed:
bigger picture on details page

whitmer committed:
basic specs

whitmer committed:
fix twitter tool to properly to user feeds

whitmer committed:
try out new canvas app store code

whitmer committed:
missing 18ns

whitmer committed:
missing app enable option

whitmer committed:
move add button to the proper place

whitmer committed:
trailing comma

whitmer committed:
more specs for third-party tools

whitmer committed:
youtube search over ssl

whitmer committed:
specs for everything except graph.tk

whitmer committed:
styling tweak

whitmer committed:
hover styling

whitmer committed:
temporary thing

whitmer committed:
youtube EDU tool

whitmer committed:
youtube edu tool

whitmer committed:
add open tapestry tool

whitmer committed:
open tapestry auth issue

whitmer committed:
ssl for twitter js

whitmer committed:
fix tool ids for data tools

whitmer committed:
include app data in reviews json

whitmer committed:
update canvas logo

whitmer committed:
fix canvas logo again

whitmer committed:
fix a couple broken tools

whitmer committed:
fix usa today tool

whitmer committed:
fix tools launched from public collections tool

whitmer committed:
longer description for wikispaces

whitmer committed:
add twitter login link near ratings area

whitmer committed:
filtering through the API

whitmer committed:
Link header for API pagination

whitmer committed:
get rid of non-working bumpin tool

whitmer committed:
correct filtering for canvas apps

whitmer committed:
speeqe icons

whitmer committed:
wolfram alpha search tool

whitmer committed:
replace 'tool' with 'app'

whitmer committed:
spec for API filtering

whitmer committed:
favicon

whitmer committed:
reword header

whitmer committed:
more index rewording

whitmer committed:
bb collaborate and non-extension launches

whitmer committed:
fix md5

whitmer committed:
fix problem with youtube traditional embeds

whitmer committed:
add generic redirect LTI tool

whitmer committed:
fix graph.tk ads redirect

whitmer committed:
youtube channel search app

whitmer committed:
app suggestion form

whitmer committed:
aleks and chalk & wire apps

whitmer committed:
fix atom feed

whitmer committed:
fix redirect tool

whitmer committed:
open tapestry and scorm cloud tools enabled

whitmer committed:
add vanilla forums app

whitmer committed:
voice thread app

whitmer committed:
WebPA app

whitmer committed:
use vimeo instead of youtube for opentapestry preview video

whitmer committed:
forgot to mark new tools as new

whitmer committed:
gooru app, refactor search results

whitmer committed:
question mark app

whitmer committed:
add clippy for configuration xml

whitmer committed:
track tool launches separately from tool loads

whitmer committed:
pending still

whitmer committed:
add no-meta parameter for api

whitmer committed:
add mh_campus, plato and question_mark apps

whitmer committed:
fix broken listing tools

whitmer committed:
re-pend apps for now

whitmer committed:
add no_meta flag for pagination

whitmer committed:
un-pend new apps

whitmer committed:
adding drupal and new york times apps

whitmer committed:
no_meta for reviews

whitmer committed:
forgot 'new' flag for nytimes

whitmer committed:
fix icons for public collections app

whitmer committed:
move app images/pages into their own folders

whitmer committed:
re-adding old icons for backwards compatibility

whitmer committed:
fix favicon dimensions

whitmer committed:
fix favicon again

whitmer committed:
fix the other favicon as well

whitmer committed:
missing file extension

whitmer committed:
Admin stuff

whitmer committed:
programmatic xml generation based on settings

whitmer committed:
fix complicated apps for new config style

whitmer committed:
fix old config links as well

whitmer committed:
remove more unnecessary templates

whitmer committed:
fix failing specs

whitmer committed:
add basic specs for xml configs

whitmer committed:
more administration, moving towards apps in db instead of filesystem

whitmer committed:
explicitly set session secret for heroku

whitmer committed:
one-line the session secret stuff

whitmer committed:
require login to submit apps

whitmer committed:
fix syntax

whitmer committed:
one-line for real

whitmer committed:
fix layout on suggestions page

whitmer committed:
fix open launch and data launch configs

whitmer committed:
re-add resource_selection for open apps

whitmer committed:
fix configs to include link text

whitmer committed:
fix failing specs, update launch urls for open apps

whitmer committed:
fix page load after login

whitmer committed:
remove active_support dependency

whitmer committed:
wrong js command for sub

whitmer committed:
clean out models code, add json generator

whitmer committed:
cleanup adming pages

whitmer committed:
switchover to db-driven lists

whitmer committed:
add test instructions for app configs

whitmer committed:
revert db loads

whitmer committed:
re-activate db lookups, fix public collections app

whitmer committed:
fix broken specs

whitmer committed:
ignore case in app sorting

whitmer committed:
fix submission js

whitmer committed:
ui tweaks

whitmer committed:
fix required option for config options

whitmer committed:
allow embedding individual tweets

whitmer committed:
cache tweet oembed lookups

whitmer committed:
fix ajax saves to work with multiple params of the same name

whitmer committed:
more submission form cleanup, allow submitting JSON datasets for data apps

whitmer committed:
explicit dimensions on images for less page jumpage

whitmer committed:
fix data json handling

whitmer committed:
updated json data documentation

whitmer committed:
fix failing config spec

whitmer committed:
change navigation text

whitmer committed:
don't lose author information on update

whitmer committed:
fix failing specs around added

whitmer committed:
add loader

whitmer committed:
add knowillage resources

whitmer committed:
update khan academy app

whitmer committed:
https routes for khan academy

whitmer committed:
change Root to All

whitmer committed:
fix js libraries for youtube upload

whitmer committed:
prulu logos and icons

whitmer committed:
let admins see config xml for pending apps

whitmer committed:
fix some placeholder text, correct logo url in admin ui

whitmer committed:
fix ampersand issues for xml configs

whitmer committed:
typo

whitmer committed:
learning catalytics images

whitmer committed:
support xml apps

whitmer committed:
text spacing in placeholders

whitmer committed:
config url replace

whitmer committed:
knowillage icons (for https)

whitmer committed:
via response app images

whitmer committed:
add Assessment category

whitmer committed:
layout tweaks, more data in admin interface

whitmer committed:
don't show description until approved

whitmer committed:
google charts app

whitmer committed:
better error message for misconfigured tool placements

whitmer committed:
update gemfile

whitmer committed:
specs for xml post fix

whitmer committed:
get rid of logos on index header

whitmer committed:
fix twitter login for non-admins

whitmer committed:
reword extensions page to not imply standards

whitmer committed:
slightly better UX on charts UI

whitmer committed:
update tool id on update

whitmer committed:
*really* fix tool id mismatch

whitmer committed:
github app

npthiery committed:
Updated instructions for ALEKS.

Updated "description" entry and added "config_directions" entry for ALEKS tool.

whitmer commented:
Since we switched to a db-backed system I've just changed this using the admin UI. I'll pull an updated backup of that for the json file as well. Sorry it took me so long to see this :-/.

npthiery commented:
Great! Thanks, Brian.

Nico

From: Brian Whitmer [mailto:notifications@github.com]
Sent: Tuesday, April 09, 2013 3:30 PM
To: whitmer/lti_example
Cc: Nicolas Thiery
Subject: Re: [lti_example] Updated instructions for ALEKS. (#1)

Since we switched to a db-backed system I've just changed this using the admin UI. I'll pull an updated backup of that for the json file as well. Sorry it took me so long to see this :-/.

—
Reply to this email directly or view it on GitHubhttps://github.com/whitmer/lti_example/pull/1#issuecomment-16144478.

npthiery commented:
Brian,

The categorization of ALEKS is currently set to “Community” and “Content”. Can it be changed to “Assessment”, “Content”, “Math”, and “Science” instead? If there is a way for me to do it, I’ll be happy to. Otherwise, thanks for making that change (and congrats on the announcement of the Canvas Apps Center!).

Thanks,

Nico

From: Brian Whitmer [mailto:notifications@github.com]
Sent: Tuesday, April 09, 2013 3:30 PM
To: whitmer/lti_example
Cc: Nicolas Thiery
Subject: Re: [lti_example] Updated instructions for ALEKS. (#1)

Since we switched to a db-backed system I've just changed this using the admin UI. I'll pull an updated backup of that for the json file as well. Sorry it took me so long to see this :-/.

—
Reply to this email directly or view it on GitHubhttps://github.com/whitmer/lti_example/pull/1#issuecomment-16144478.
